examples/client.c

```diff
@@ -770,29 +773,36 @@ usage( const char *program, const char *version) {
      "\tPKCS11 URI file definitions have to be in DER, not PEM, format.\n"
      "\tOtherwise all of '-c certfile', '-j keyfile' or '-C cafile' are in\n"
      "\tPEM format.\n\n"
-     "\t-c certfile\tPEM file or PKCS11 URI for the Certificate. The private\n"
-     "\t       \t\tkey can be in the PEM file, or use the same PKCS11 URI.\n"
-     "\t       \t\tIf not, the private ney is defined by '-j keyfile'\n"
+     "\t-c certfile\tPEM file or PKCS11 URI for the certificate. The private\n"
+     "\t       \t\tkey can also be in the PEM file, or has the same PKCS11\n"
+     "\t       \t\tURI. If not, the private key is defined by '-j keyfile'\n"
      "\t-j keyfile\tPEM file or PKCS11 URI for the private key for the\n"
-     "\t       \t\tnertificate in '-c certfile' if the parameter is different\n"
-     "\t       \t\tfrom certfile in '-c certfile'\n"
-     "\t-C cafile\tPEM file or PKCS11 URI for the CA Certificate that was\n"
-     "\t       \t\tused to sign the certfile. This will trigger\n"
-     "\t       \t\tthe validation of the server certificate.  If certfile is\n"
-     "\t       \t\tself-signed (as defined by '-c certfile'), then you need\n"
-     "\t       \t\tto have on the command line the same filename for both\n"
-     "\t       \t\tthe certfile and cafile (as in '-c certfile -C certfile')\n"
-     "\t       \t\tto trigger validation\n"
+     "\t       \t\tcertificate in '-c certfile' if the parameter is\n"
+     "\t       \t\tdifferent from certfile in '-c certfile'\n"
+     "\t-n     \t\tDisable remote peer certificate checking\n"
+     "\t-z     \t\tDisable checking that server and client certificates\n"
+     "\t       \t\twere signed by a common CA (used for mutual\n"
+     "\t       \t\tauthentication)\n"
+     "\t-C cafile\tPEM file or PKCS11 URI for the CA certificate that was\n"
+     "\t       \t\tused to sign the client certfile. The contents of cafile\n"
```

Comment:
On the client side, isn't it the "server certificate", which is intended to be signed by the clients CA?

Reply:
Comment corrected